include/pmacc/math/complex/Bessel.tpp

```diff
@@ -86,19 +86,23 @@ namespace bessel
     >
     struct Cbesselj0Base
     {
-        using result = pmacc::math::Complex< T_Type >;
+        using Result = pmacc::math::Complex< T_Type >;
         using complex_T = pmacc::math::Complex< T_Type >;
```

Comment:
actually it is the same for the other types as well. Always keep types with an upper case letter

Reply:
In an offline discussion with @psychocoderHPC we agreed to not adjust the other types. `float_T` and `complex_T` are more consistent with the original `float_X ` and `float_64` naming.

Reply:
all right, ok with me! :)